src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -567,23 +565,31 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
                 continue;
             }
 
-            // We get the name and the path to graph as a pair separated by |.
+            // We get the name .
             char topic_name[FRONTEND_DATA_LENGTH];
             bzero(topic_name, FRONTEND_DATA_LENGTH + 1);
-
             read(connFd, topic_name, FRONTEND_DATA_LENGTH);
 
+            string con_message = "successfully get kafka topic";
+            int con_result_wr = write(connFd, con_message.c_str(), con_message.length());
+            if (con_result_wr < 0) {
+                frontend_logger.log("Error writing to socket", "error");
+                loop = true;
+                continue;
+            }
+
+            Utils utils;
             string topic_name_s(topic_name);
+            frontend_logger.log("sandaruwan " , "info");
```

Comment:
Why do we have this log line here?

Reply:
For the testing purpose I used it. Now it doesn’t need sir.